### PR TITLE
Feature/fix limitations

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -12,5 +12,3 @@ yarn-error.log*
 
 # Unit test / coverage reports
 coverage/
-
-docs/
```
Comment (severinbeauvais) [inline .gitignore:16]: Note to other devs: although this is checked in, we can ignore anything in docs/ during PR review.

File: src/components/DetailComment/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcrs-shared-components/detail-comment",
-    "version": "1.0.3",
+    "version": "1.0.5",
```
Comment (severinbeauvais) [inline src/components/DetailComment/package.json:3]: What rules should we implement for updating the version (or does Lerna do it for us)?

Here, the "patch" number is changes, not the minor version. I am wondering when should update the major/minor/patch versions... Thoughts?

**Review by severinbeauvais — APPROVED** [review]
Looks good. I'm glad this wasn't too complicated in the end :)

Note to other devs: do not PR from your fork/branch to this project/master. Instead, create a branch in this repo and PR from that branch to master; this will allow proper up-versioning of components.

If anyone has questions about what to do, please post them and we will update the "how to" docs.

File: src/components/DetailComment/package.json (hunk 2 above)
Comment (vysakh-menon-aot) [reply src/components/DetailComment/package.json:3]: I missed to merge last version update(1.0.4 is published in npm before), so i choose next patch version. We can follow [npm standard](https://docs.npmjs.com/about-semantic-versioning)

**Pull request merged**